test/tests/performance/pbs_equiv_classes_perf.py

```diff
@@ -177,94 +160,109 @@ def test_server_queue_limit(self):
+            J = Job(TEST_USER1, attrs=a)
+            for y in range(100):
+                self.server.submit(J)
 
-        c = self.scheduler.cycles(lastN=1)[0]
-        cycle1_time = c.end - c.start
+        # Get time for ~500 classes
+        cyc5 = self.get_cycle_time()
 
-        t = time.time()
+        # Submit jobs as pbsuser1 from tstgrp02 in workq
+        for x in range(100):
+            a = {'Resource_List.select': '1:ncpus=1',
+                 'Resource_List.walltime': int(x),
+                 'group_list': TSTGRP2, ATTR_q: 'workq'}
+            J = Job(TEST_USER1, attrs=a)
+            for y in range(100):
+                self.server.submit(J)
 
-        # Rerun a scheduling cycle
-        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
-                            {'scheduling': 'True'})
-        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
-                            {'scheduling': 'False'})
+        # Get time for 60k jobs for ~600 classes
+        cyc6 = self.get_cycle_time()
 
-        # Wait for cycle to finish
-        m = self.scheduler.log_match("Leaving Scheduling Cycle",
-                                     starttime=int(t),
-                                     max_attempts=200)
-        self.assertTrue(m)
+        # Submit jobs as pbsuser2 from tstgrp01 in workq
+        for x in range(100):
+            a = {'Resource_List.select': '1:ncpus=1',
+                 'Resource_List.walltime': int(x),
+                 'group_list': TSTGRP1, ATTR_q: 'workq'}
+            J = Job(TEST_USER2, attrs=a)
+            for y in range(100):
+                self.server.submit(J)
 
-        c = self.scheduler.cycles(lastN=1)[0]
-        cycle2_time = c.end - c.start
+        # Get time for 70k jobs for ~700 classes
+        cyc7 = self.get_cycle_time()
 
-        self.logger.info('Cycle 1: %d Cycle 2: %d' %
-                         (cycle1_time, cycle2_time))
-        self.assertTrue(cycle1_time >= cycle2_time)
+        # Submit jobs as pbsuser2 from tstgrp03 in workq
+        for x in range(100):
+            a = {'Resource_List.select': '1:ncpus=1',
+                 'Resource_List.walltime': int(x),
+                 'group_list': TSTGRP3, ATTR_q: 'workq'}
+            J = Job(TEST_USER2, attrs=a)
+            for y in range(100):
+                self.server.submit(J)
+
+        # Get time for 80k jobs for ~800 classes
+        cyc8 = self.get_cycle_time()
+
+        # Print the time taken for all the classes and compare
+        # it against previous releases
+        self.logger.info("time taken for 100 classes is %d,\
+                         200 classes is %d,\
+                         300 classes is %d,\
+                         400 classes is %d,\
+                         500 classes is %d,\
+                         600 classes is %d,\
+                         700 classes is %d,\
+                         800 classes is %d"
+                         % (cyc1, cyc2, cyc3, cyc4, cyc5, cyc6, cyc7, cyc8))
```

Comment:
With the string continuation with the \\, won't the white space be taken as literal white space?  I think you want to use multiple quoted strings.  You can use "a"     "b" == "ab".  Also, some of your jobs may run.  By the time you reach the bottom of your 80,000 job test, you won't have all 80,000 jobs any more.  Some will have completed.  In test_basic() I made sure the jobs couldn't run.  You may want to do that here.